app/Http/Controllers/Adm/Mship/Feedback.php

```diff
@@ -12,6 +13,44 @@
 
 class Feedback extends \App\Http\Controllers\Adm\AdmController
 {
+    public function getNewForm()
+    {
+        $question_types = Type::all();
+        $new_question = new Question();
+
+        return $this->viewMake('adm.mship.feedback.new')
+            ->with('question_types', $question_types)
+            ->with('new_question', $new_question);
+    }
+
+    public function postNewForm(UpdateFeedbackFormRequest $request)
+    {
+        $new_ident = isset($_POST['ident']) ? $_POST['ident'] : null;
```

Comment:
Ideally should be using `$request->input('ident')` here, and same for line below